 frequently_asked_question_bot/bot/run.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from dff.messengers.telegram import PollingTelegramInterface
+from dff.pipeline import Pipeline
+
+from dialog_graph import graph
+from pipeline_services import pre_services
+
+
+def get_pipeline(interface_cli=None) -> Pipeline:
```

Comment:
```suggestion
def get_pipeline(interface_cli: bool = False) -> Pipeline:
```
Agreed with Roman

Reply:
We might also want to rename this parameter to use_cli or something else that is more suitable for a boolean variable